app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,23 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+ 
+  def perform(initiator, benchmark_type, benchmark_result_type, category)
+    commit_objects = BenchmarkRun.where(benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).offset(10).limit(1000)
+
+    commit_results = Array.new
+    commit_objects.drop(1).each do |object|
+      commit_results << object.result[category]
+    end
+    
+    results_avg = commit_results.inject{ |sum, el| sum + el }.to_f / commit_results.size
+    results_standard_deviation = Math.sqrt(commit_results.inject(0){|accum, i| accum +(i-results_avg)**2 }/(commit_results.length - 1).to_f)
+    results_param = results_avg + 2*results_standard_deviation
+
+    current = BenchmarkRun.find_by(initiator: initiator)
```

Comment:
Hmm I don't quite understand what you mean.

`BenchmarkRun.find_by(initiator: initiator)` will fetch all benchmark runs that belongs to a particular commit